modules/salt/pages/formula-pxe.adoc

```diff
@@ -8,10 +8,14 @@ The PXE formula is used to configure PXE booting on the branch server.
 . In the {susemgr} {webui}, open the details page for the branch server, and navigate to the [guimenu]``Formulas`` tab.
 . Select the [systemitem]``Pxe`` formula, and click [btn]``Save``.
 . Navigate to the menu:Formulas[Pxe] tab, and set these parameters:
-* In the [guimenu]``Kernel filename`` field, keep the default value.
-* In the [guimenu]``Initrd filename`` field, keep the default value.
-* In the [guimenu]``Kernel command line parameters`` field, keep the default value.
-    For more information about possible values, see <<retail.sect.formulas.pxe.kernelparams>>.
+* In the [guimenu]``Kernel Filename`` field, keep the default value.
+* In the [guimenu]``Initrd Filename`` field, keep the default value.
+* If the terminals connecting to this branch server are running {arm64} architecture, check the [guimenu]``Enable ARM64 UEFI boot`` box.
```

Comment:
I'd also expect "boot" (in the menu string) to be capitalized.

Reply:
OK, I changed it to "Boot" here https://github.com/SUSE/salt-formulas/pull/106

Also I am not sure if it is clear enough that this affects only arm64 terminals booting from network. Raspberry Pi is also arm64 architecture, but it boots from SD card, so this checkbox has no effect on it.

Reply:
```suggestion
* If the terminals connecting to this branch server are running {arm64} architecture and booting from the network, check the [guimenu]``Enable ARM64 UEFI boot`` box.
```

Reply:
Looks good.